src/app/tests/suites/certification/Test_TC_DIAG_TH_NW_1_2.yaml

```diff
@@ -248,35 +241,35 @@ tests:
       attribute: "PartitionId"
       response:
           constraints:
-              type: uint32
+              type: null
```

Comment:
Does not make sense.  I'm going to stop commenting on these, but there are a lot....  Please undo all these contraint type changes.